db/academyPortal_2019-10-02.sql

```diff
@@ -202,6 +202,25 @@ CREATE TABLE `hiring_partner_companies` (
   `postcode` varchar(255) DEFAULT NULL,
   `phone_number` varchar(255) DEFAULT NULL,
   `url_website` varchar(255) DEFAULT NULL,
+  `primary_contact` int(11) unsigned DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  KEY `primary_contact` (`primary_contact`),
+  CONSTRAINT `hiring_partner_companies_ibfk_1` FOREIGN KEY (`primary_contact`) REFERENCES `hiring_partner_contacts` (`id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
```

Comment:
it is required for a hiring partner to have a primary contact. is this covered by this table?

Reply:
My interpretation is that it is required for a contact to be a primary contact, or not.